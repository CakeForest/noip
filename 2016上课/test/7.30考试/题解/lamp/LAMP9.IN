300
0 200 181 9578
1 170 26 8911
1 73 100 8857
1 35 56 5695
2 183 23 1459
1 29 24 776
2 4 14 5547
5 56 16 1300
1 43 84 6968
5 41 4 9294
2 131 8 722
1 133 78 8729
5 141 22 4987
1 143 67 9320
11 41 6 8395
9 163 42 4209
4 57 39 4990
13 193 14 7465
15 177 6 5299
6 14 8 7307
1 92 54 1797
15 37 5 223
18 103 14 3401
5 73 5 2854
7 167 6 2048
6 38 5 738
3 141 91 5622
2 58 17 8928
2 85 7 1009
28 69 1 7189
10 165 2 9616
20 90 4 5126
17 158 5 785
16 164 40 8247
3 156 21 6948
1 30 94 7829
25 198 3 3550
32 39 2 3585
17 124 9 7521
1 74 72 1698
32 65 1 5876
25 70 5 9908
4 189 48 1582
42 18 1 9102
10 184 2 2760
25 133 2 5464
42 28 4 4168
16 176 24 1709
34 16 15 2446
23 110 13 8514
37 82 2 9324
39 171 7 947
17 103 14 5623
29 55 4 7723
17 122 11 3937
46 178 1 6570
46 64 1 5112
14 24 2 4946
32 189 3 3542
36 36 56 5034
30 162 1 4618
56 165 1 3375
22 80 1 4923
2 25 10 1901
32 28 1 7652
7 77 4 6514
41 13 1 2875
11 95 4 5006
8 151 5 208
53 20 1 5956
50 44 5 952
15 169 5 7586
49 129 1 3300
51 0 2 8422
57 46 1 8556
13 102 8 3100
13 83 20 3938
45 108 2 5046
74 61 2 9212
45 103 2 5959
27 30 90 5658
75 124 1 8355
4 2 31 7359
42 22 5 4886
11 27 3 2168
53 186 11 4929
74 124 2 3520
78 98 1 8113
78 68 2 245
58 118 1 4200
31 57 1 177
84 169 3 2148
31 150 1 430
52 21 5 6366
17 3 10 3206
74 143 1 8330
37 98 2 6662
50 128 12 4880
56 89 1 6216
33 140 1 3873
39 153 5 8405
3 121 46 1496
46 89 1 7623
41 81 1 6644
41 127 1 1386
84 111 1 4322
101 109 3 2824
60 157 21 5373
26 109 2 5531
26 78 3 1868
43 27 15 6288
98 145 8 9523
97 150 2 1187
46 178 2 8545
82 106 1 938
94 1 4 5536
14 97 35 7235
89 106 2 9589
47 157 1 4641
32 99 2 8676
84 194 2 7173
95 127 6 5877
12 79 20 6161
73 151 1 2189
42 45 2 1783
35 172 11 1603
64 147 5 5430
118 173 1 7989
21 146 40 9454
11 183 4 291
77 76 1 5455
27 123 73 5603
111 164 3 7767
55 140 7 7095
75 36 1 5728
89 199 1 8582
15 198 5 2439
23 132 2 7877
93 34 1 9472
117 3 26 107
80 186 1 2205
91 106 1 8015
104 65 1 9566
37 155 3 1993
5 83 15 752
127 65 5 6422
90 13 1 499
34 190 24 3836
100 1 1 9276
42 137 3 2732
57 123 1 6193
105 95 1 1487
135 109 1 5777
43 106 6 8134
120 35 2 6034
129 129 10 6606
79 96 1 9672
72 4 1 2813
29 176 2 3967
142 89 1 1489
126 0 11 2823
150 161 1 5476
30 121 1 7985
98 51 8 5904
74 194 1 579
48 126 6 2922
97 109 1 9865
158 164 1 5143
76 135 5 8917
49 45 1 133
55 129 2 7494
50 111 10 1014
92 85 3 6284
168 103 1 3850
104 163 1 4619
7 36 4 9995
89 180 2 1227
21 26 8 5104
68 193 2 5385
69 123 4 5260
162 71 1 9406
1 53 45 8668
157 173 1 3478
152 50 1 1803
104 128 1 6097
3 160 18 9910
133 157 1 8242
95 98 9 6378
11 0 2 2748
94 155 1 1999
12 2 28 2595
89 121 2 6469
34 141 25 8722
185 30 1 8943
126 16 3 103
94 185 5 5014
177 102 2 1781
97 179 2 2427
118 185 1 1039
19 64 3 99
101 179 4 3438
189 79 2 2922
121 150 1 9788
172 3 5 1342
137 29 4 7747
75 30 1 8022
133 159 1 8430
203 138 1 7184
190 45 1 4640
131 156 1 3491
104 165 1 3848
12 184 15 9811
126 94 5 1392
64 80 10 4988
104 106 1 8205
45 180 2 6913
61 19 1 8324
29 45 4 927
111 129 9 5137
28 105 12 2018
85 103 3 7319
181 111 1 9449
28 189 16 6498
190 145 1 7602
37 95 3 4259
183 7 1 4983
33 7 4 7012
216 31 1 7702
138 198 2 8970
86 48 8 2344
65 188 1 6660
192 40 2 9255
184 21 1 8773
37 115 2 8855
131 104 1 3161
22 49 4 3867
178 91 1 4230
236 98 3 7456
152 140 1 9386
56 65 1 4248
203 133 1 8179
170 197 1 6265
65 99 1 3044
42 5 4 2024
59 104 1 1659
6 90 8 2567
111 174 7 8487
61 149 1 6137
12 197 61 2811
14 165 52 1686
33 80 1 2977
209 45 1 3568
29 4 6 7348
215 8 1 3732
104 149 1 2268
162 87 1 6589
148 43 6 1698
235 159 1 3670
165 188 1 2824
68 149 3 3269
170 43 1 2398
213 131 4 2685
230 95 1 545
206 80 1 447
82 2 1 7851
64 83 10 1323
150 48 2 8323
64 70 3 897
198 194 1 1023
20 171 4 7362
109 45 2 7281
88 68 1 8905
227 148 2 4185
111 141 9 3890
230 13 3 9038
219 71 9 965
13 187 8 9288
155 8 1 5556
152 77 1 8844
77 71 8 826
91 129 1 63
130 176 1 3469
239 194 1 1928
119 65 1 602
15 151 3 4456
13 33 8 1696
11 147 2 7900
227 127 1 2166
49 12 9 8459
129 6 4 855
272 103 1 554
230 147 2 5547
248 47 1 4973
61 78 1 6745
66 132 2 2918
138 25 2 3541
143 114 1 3756
221 164 3 4500
92 89 2 9893
45 137 1 4273
