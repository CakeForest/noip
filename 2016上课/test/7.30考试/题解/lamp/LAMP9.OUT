7 47212
1 4 6 20 36 83 182
